spec.md

```diff
@@ -649,6 +657,20 @@ message CreateVolumeRequest {
   // If specified, the new volume will be pre-populated with data from
   // this source. This field is OPTIONAL.
   VolumeContentSource volume_content_source = 6;
+  
+  // Specifies where (regions, zones, racks, etc.) the provisioned
+  // volume MUST be accessible from.
+  // An SP SHALL advertise the requirements for topological
+  // accessibility information in documentation. COs SHALL only specify
+  // topological accessibility information supported by the SP. COs
+  // SHALL permit passing through topological accessibility information.
+  // This field is OPTIONAL.
+  // This field MAY be specified only if the SP has the
```

Comment:
This field SHALL NOT be specified unless the SP has the ACCESSIBILITY_CONSTRAINTS plugin capability.